SwiftWisdom/Core/StandardLibrary/NSAttributedString/NSMutableAttributedString+Format.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-extension NSMutableAttributedString {
+public extension NSMutableAttributedString {
```

Comment:
the doc says "If you extend a public or internal type, any new type members you add have a default access level of internal." - so I'm not actually so sure on this note. Let's not worry about this for now, just merge in your changes, we will test before a merge.